internal/functions/efgs/download-batch.go

```diff
@@ -98,20 +98,27 @@ func publishAllKeys(ctx context.Context, config *config, keys []efgsapi.Diagnosi
 
 	logger.Infof("Sorted downloaded keys into %v groups (countries)", len(sortedKeys))
 
+	var errors []string
+
 	for country, countryKeys := range sortedKeys {
 		haid, exists := config.HaidMappings[country]
 		if !exists {
-			return fmt.Errorf("HAID mapping for %v doesn't exist", country)
+			errors = append(errors, fmt.Sprintf("Keys from %v were provided but HAID mapping doesn't exist!", country))
+			continue
 		}
 
 		logger.Debugf("Uploading %v keys from %v to our Key server", len(countryKeys), country)
```

Comment:
It should be `logger.Infof...` like on the line 99. Or why it is on another debug level?

Reply:
No problem with that. As you wish.